blueocean-web/src/main/js/main.jsx

```diff
@@ -146,3 +146,6 @@ function startApp(routes, stores) {
 Extensions.store.getExtensions(['jenkins.main.routes', 'jenkins.main.stores'], (routes = [], stores = []) => {
     startApp(routes, stores);
 });
+
+// Enable page reload.
+require('./reload');
```

Comment:
In reply to @cliffmeyers comment in the code ... yes, I was going to do an `import` but then I thought that would just look weird without doing it as you described i.e. adding an `export` in `reload.js`. Based on that I just decided to do the `require` Vs adding superfluous fluff. I'm fine with it tbh and I know it was just a Nit for you..